src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const fixedRobot = {};
+  const robotKey = Object.values(robot);
+  const robotValue = Object.keys(robot);
```

Comment:
even if you DO need to use it, let's give it proper names
```suggestion
  const robotKeys = Object.values(robot);
  const robotValues = Object.keys(robot);
```

Reply:
however, it'd be better to use simple for-in-loop — consistent and readable